src/Microsoft.ML.Analyzer/Microsoft.ML.Analyzer.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeAnalyzerInPackage>Microsoft.ML</IncludeAnalyzerInPackage>
+    <RunApiCompat>true</RunApiCompat>
```

Comment:
Do you actually ship this analyzer?  Also changing targetFramework is a breaking change since it drops compatibility with what you previously targeted.

Reply:
Yes we ship this analyzer.

But this project has no public APIs, so I don't believe we need to Run API Compat on it.

---
In reply to: [279938491](https://github.com/dotnet/machinelearning/pull/3623#discussion_r279938491) [](ancestors = 279938491)

Reply:
Should the targetFramework remain 1.3 @eric.erhardt@microsoft.com?
I'll remove the `RunApiCompat`.

---
In reply to: [279940309](https://github.com/dotnet/machinelearning/pull/3623#discussion_r279940309) [](ancestors = 279940309,279938491)

Reply:
I wouldn't change the TFM unless absolutely necessary. I think we should be able to revert this file.